tikv-client/src/main/java/com/pingcap/tikv/types/TimeType.java

```diff
@@ -31,6 +31,7 @@
 // https://dev.mysql.com/doc/refman/8.0/en/time.html
 
 public class TimeType extends DataType {
+  public static final TimeType TIME = new TimeType(MySQLType.TypeDuration);
```

Comment:
why do we create a time type here?

Reply:
We need a basic type for typemap and generator

Reply:
gotcha.